introduction.md

```diff
@@ -1 +1,6 @@
 My name is TylerMays. My favorite Code School path is JavaScript. I am not a github master, but I am here to learn.
+Skills:
+*HTML
+*CSS
+*JavaScript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?